packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1442,4 +1463,74 @@
     <protected>false</protected>
     <shortDescription>Show less</shortDescription>
   </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionTopic</fullName>
+    <value>Is the answer about the right topic?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Question Topic</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionHallucination</fullName>
+    <value>Is the answer free of hallucinated content?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Question Hallucination</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionDocumented</fullName>
+    <value>Can the question be answered by the documentation?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Question Documented</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionReadable</fullName>
+    <value>Is the answer readable?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Question Readable</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackNotSure</fullName>
+    <value>Not sure</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Answer Not Sure</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackAnswerEvaluation</fullName>
+    <value>Answer evaluation</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Answer Evaluation</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackDocumentUrl</fullName>
+    <value>Link to correct answer</value>
```

Comment:
```suggestion
    <value>Link to the correct answer</value>
```

Reply:
That's coming from the mockup, not my decision.